src/smo/data/tuplet.ts

```diff
@@ -19,16 +162,15 @@ import { getId, SmoAttrs, Clef } from './common';
  * @category SmoParameters
  */
 export interface SmoTupletParams {
-  notes: SmoNote[],
-  id?: string,
   numNotes: number,
+  notesOccupied: number,
```

Comment:
More of a question: is notesOccupied something that can't be calculated by tuplet.ts?  I didn't really like the way tuplets are represented in vexflow - I'd prefer a numNotes and an overall duration.  I think this still makes sense - isn't notesOccupied always just this.totalTicks / this.stemTicks?  Because I'd prefer not to have value parameters when a value can be calculated from other value parameters.

Reply:
I need to double-check this. I think I wanted to have a similar or the same interface as VexFlow. When I was looking at how this is handled in XML, VexFlow and now Smoosic each had a different approach to naming....